tests/integration/postgres/test_postgres_target_adapter.py

```diff
@@ -156,10 +159,13 @@ def test_restore_data_from_shared_replica(docker_flush):
         envars=['POSTGRES_USER=snowshu',
                 'POSTGRES_PASSWORD=snowshu',
                 'POSTGRES_DB=snowshu',
-                f'PGDATA={DOCKER_REPLICA_MOUNT_FOLDER}'])
+                f'PGDATA=/pgdata'])
```

Comment:
should it be in uppercase?

Reply:
Does not matter, it's a dir name. Default ones under "/" are lower case

Reply:
just a naming convention to follow

Reply:
DOCKER_REPLICA_MOUNT_FOLDER evaluates to "/app/snowshu_replica_data_shared", so it is still lower case. You might be confusing global variable name (all upper) to the path itself (generally lower). The change was made since pgdata content is no longer inside the volume (only the dump is).